crates/database/src/settlement_scores.rs

```diff
@@ -0,0 +1,64 @@
+use {crate::auction::AuctionId, bigdecimal::BigDecimal, sqlx::PgConnection};
+
+#[derive(Debug, PartialEq, sqlx::FromRow)]
+pub struct Score {
+    pub auction_id: AuctionId,
+    pub winning_score: BigDecimal,
+    pub reference_score: BigDecimal,
+    pub block_deadline: i64,
+}
+
+pub async fn insert(
+    ex: &mut PgConnection,
+    auction_id: AuctionId,
+    winning_score: BigDecimal,
+    reference_score: BigDecimal,
+    block_deadline: i64,
```

Comment:
Don't we want to just pass the score struct into this method and unpack it inside the function with dot notation?

Reply:
Done.